whipper/test/test_common_mbngs.py

```diff
@@ -44,6 +44,25 @@ def testTrackTitle(self):
         track1 = metadata.tracks[0]
         self.assertEqual(track1.title, 'Brownsville Turnaround')
 
+    def testClassicalTrack(self):
+        """
+        Test if composers and performers are extracted properly
+        see https://github.com/whipper-team/whipper/issues/191
+        """
+        # Using: Début Recital - Martha Argerich
+        # https://musicbrainz.org/release/5598dd88-5a14-4661-a9cc-fe37a5246028
+        filename = 'whipper.release.5598dd88-5a14-4661-a9cc-fe37a5246028.json'
+        path = os.path.join(os.path.dirname(__file__), filename)
+        with open(path, "rb") as handle:
+            response = json.loads(handle.read().decode('utf-8'))
+
+        metadata = mbngs._getMetadata(response['release'], classical=True)
+        track1 = metadata.tracks[0]
+        self.assertEqual(track1.composers, ['Fryderyk Chopin'])
```

Comment:
What happens for a Recording with multiple Works of different composers? Or a work with multiple composers?

Reply:
Maybe i should choose a better test case for this.

Reply:
I have added a brand new test case for this. Thanks :D